City-of-Helsinki/berth-reservations-admin
# VEN-769 | Add ability to create boats for customers

## Description :sparkles:

Adds `BoatCreateForm` which allows for the creation of new boats.

### Closes :no_good_woman:

[VEN-769](https://helsinkisolutionoffice.atlassian.net/browse/VEN-769)

## Testing :alembic:

### Manual testing :construction_worker_man:

- Open a customer's page
- An "Add boat" button should be present in the customer's boats card
- It should be possible to create a new boat by submitting the form with correctly validated values

## Additional notes :spiral_notepad:

"Fixme" changes in `src/features/customerView/utils.ts` and `src/features/customerView/queries.ts` are addressed in a separate PR.
